dev/build_release.py

```diff
@@ -287,11 +288,32 @@ def start_subsystem_build(self, name):
     # Note: 'candidate' is just the gradle task name. It doesn't indicate
     # 'release candidate' status for the artifacts created through this build.
     return BackgroundProcess.spawn(
-      'Building and publishing {name}...'.format(name=name),
+      'Building and publishing Debian for {name}...'.format(name=name),
       'cd "{gradle_root}"; ./gradlew {extra} {target}'.format(
           gradle_root=gradle_root, extra=' '.join(extra_args), target=target)
     )
 
+  def start_container_build(self, name):
+    """Start a subprocess to build a container image of the subsystem.
+
+    Uses Google Container Builder with configuration files produced
+    during BOM generation to build the container images. The configuration
+    files are assumed to be in the parent directory of the subsystem's
+    Gradle root.
+
+    Args:
+      name [string]: Name of the subsystem repository.
+
+    Returns:
+      BackgroundProcess
+    """
+    gradle_root = self.determine_gradle_root(name)
+    return BackgroundProcess.spawn(
+      'Building and publishing container image for {name}...'.format(name=name),
```

Comment:
That's a good point - the only problem is that each registry (that has a builder) has slightly different semantics for triggering a build. We could provide an optional local docker builder that would work for any registry. i.e. if gcloud isn't installed `docker build -t <name> .` in the project's directory does the trick.